packages/flutter_driver/lib/src/extension.dart

```diff
@@ -243,8 +260,8 @@ class FlutterDriverExtension {
 
     _prober.binding.hitTest(hitTest, startLocation);
     _prober.binding.dispatchEvent(pointer.down(startLocation), hitTest);
-    await new Future<Null>.value();  // so that down and move don't happen in the same microtask
-    for (int moves = 0; moves < totalMoves; moves++) {
+    await new Future<Null>.value(); // so that down and move don't happen in the same microtask
+    for (int moves = 0; moves < totalMoves; moves += 1) {
```

Comment:
The problem with `++` is that in some situations, the postfix operator is less efficient than `+= 1`. Since loops are often performance critical, people get all worried about this and end up writing `++moves`. And the prefix operator is super ugly and horrible. So I'm on a mission to entirely kill the ++ and -- operators. :-)